docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  admin_server:
+    build:
+      dockerfile: ./admin_server/Dockerfile
+    ports:
+      - "8080:8080"
+    environment:
+      - SPRING_PROFILE=production
+
+  facial_recognition:
+    build:
+      dockerfile: ./facial_recognition/Dockerfile
+    ports:
+      - "9090:9090"
+    environment:
+      - PYTHON_PROFILE=production
```

Comment:
There should be a Jira ticket to update these environment variables to what the flask application needs.